app/services/flow/flow_state_machine.rb

```diff
@@ -169,17 +169,13 @@ def flow_finish
       redirect_to send(@final_url)
     end
 
-    def redirect_to_step(step)
+    def redirect_to_step(_step)
```

Comment:
Should `_step` be `next_step`? I will come back to this a dig in more

Reply:
This is from the original delete branch and here's why I think the change was made - `step` was [removed](https://github.com/18F/identity-idp/pull/11324/files/b1d8e1893974a9492ad1ff56e7a5b1da8eddc9f9#diff-af7c4843eeac525a29ad901beaba63d16aec567cd7d39b5409b345fbfaccbdecL174) from this method, so it's no longer being used. I think the original author added the underscore prefix to indicate that it's not being used in the method.